app/assets/javascripts/edit/id.js.erb

```diff
@@ -10,7 +10,7 @@ $(document).ready(function () {
     var params = {};
 
     if (mapParams.object) {
-      params.id = mapParams.object.type[0] + mapParams.object.id;
+      params.id = mapParams.object.type + '/' + mapParams.object.id;
```

Comment:
FYI: this also resolves a bug which was previously causing unrelated OSM nodes from being loaded when the user had a no**t**e open (e.g. when opening `osm.org/edit?note=1`, the built in iD would load `node/1` instead).

To fix that, this is now using a lightly different syntax for the hash parameter `id` (see [docs](https://github.com/openstreetmap/iD/blob/develop/API.md#url-parameters)).